infra/packages/shared/src/index.ts

```diff
@@ -42,3 +42,101 @@ export const CLOUDWATCH_KINESIS_STREAM_ROLE_ARN =
 export { getMacroApiToken } from './macro_api_token';
 export { getMacroNotify } from './macro_notify';
 export { getSearchEventQueue } from './search_event_queue';
+
```

Comment:
I'd prefer if all of this was put into a dedicated `service_urls.ts` then re-exported in this index.ts. See above for similar use cases.